layouts/partials/head/others.html

```diff
@@ -8,7 +8,7 @@
 {{ $utils := resources.Get "dist/utilities.css" }}
 {{ $css := slice $scss $utils | resources.Concat "dist/style.css" }}
 
-{{ if .Site.IsServer }}
+{{ if .hugo.IsServer }}
```

Comment:
https://gohugo.io/functions/hugo/isserver/

```suggestion
{{ if hugo.IsServer }}
```